wtprompt/core.py

```diff
@@ -32,6 +34,17 @@ def _get_prompt_text(self, prompt_name: str):
     def get_prompts(self):
         return self._prompts
 
+    def save_prompt_report(self, outfile: str):
+        # Hashing prompts
+        prompt_hashes = {}
+        for key, prompt in self._prompts.items():
+            prompt_hashes[key] = hashlib.sha256(prompt.encode("utf-8")).hexdigest()
+
+        # Saving report
+        outfile = pathlib.Path(outfile).with_suffix('.json')
```

Comment:
nitpicking: I personally prefer letting the user choose whatever extension they want.

If they want a `.json` they can just call this with `outfile` ending with that suffix.